### ISSUE TITLE
Missing VCRUNTIME140.dll and MSVCP140.dll

### ISSUE THREAD

**Issue by frantz** [agocke/xres2lnk#1]
I have system errors when running `xres2lnk.exe` because of missing `VCRUNTIME140.dll` and `MSVCP140.dll`

I've installed Microsoft Visual C++ 2015 redistributable (x64), but have had no more success.

Can you help me? ( I'm running win10 with creator update and I'm not that much familiar with Windows :P )

**Comment by agocke** [comment]
@frantz Sorry about the delay! I've been out of the country and just saw your comment. This was my mistake -- I thought I was statically compiling the standard library so you wouldn't need any dependencies, but it turns out I was not.

I've created a new Release to hopefully fix the issue here: https://github.com/agocke/xres2lnk/releases/tag/v1.1.1

Also, I suspect the reason why downloading the redistributable didn't work is that I compiled the program as x86, since that works on almost all machines.
